plugins/steve/actions.js

```diff
@@ -1,11 +1,26 @@
 import https from 'https';
 import cloneDeep from 'lodash/cloneDeep';
 import { SCHEMA } from '@/config/types';
+import { createYaml } from '@/utils/create-yaml';
 import { normalizeType } from './normalize';
 import { proxyFor, SELF } from './resource-proxy';
 
 export default {
-  request({ dispatch }, opt) {
+  async request({ dispatch, rootGetters }, opt) {
+    // Handle spoofed types instead of making an actual request
+    // Spoofing is handled here to ensure it's done for both yaml and form editing.
+    // It became apparent that this was the only place that both intersected
+    if (opt.url.includes('spoofed')) {
```

Comment:
This is going to erroneously fire if you try to edit any (regular) resource with a name that contains `spoofed`.  Granted unlikely, but would prefer this (and the other places `/spoofed{api}` is made a URL be based on a single constant, and contain some character that is for sure not part of a legitimate type or id.

Reply:
Switched to __[[spoofed]]__ and __[[spoofedapi]]__.